lnrpc/routerrpc/router.proto

```diff
@@ -176,6 +176,24 @@ service Router {
     */
     rpc UpdateChanStatus (UpdateChanStatusRequest)
         returns (UpdateChanStatusResponse);
+
+    /*
+    XAddLocalChanAliases is an experimental API that creates a set of new
+    channel SCID alias mappings. The list of successfully created mappings is
+    returned. This is only a locally stored alias, and will not be communicated
+    to the channel peer via any message. Therefore, routing over such an alias
+    will only work if the peer also calls this same RPC on their end. If an
```

Comment:
You are correct, for _just_ the routing part it's enough if only one side sets this. But for the custom channel functionality (especially the RFQ part where the custom data within an HTLC might reference such an SCID alias) the alias needs to be known to both parties to fully work.

We added the X because of the requirement of both nodes needing to call this individually and it not being negotiated over a custom p2p message between the peers. I think it's still the goal to eventually (not this PR) make it possible to communicate this over the wire, that's when we're going to remove the X.